### PR TITLE
feat(new wallet): require alphanumeric lightning addresses

### PR DESCRIPTION
**Pull request by bumi**
image

### CODE REVIEW

File: src/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
           },
           "optional_lightning_address_label": "Choose your Lightning Address (optional)",
           "optional_lightning_address_suffix": "@getalby.com",
+          "optional_lightning_address_title": "numbers and letters, at least 3 characters",
```
Comment (escapedcat) [inline src/i18n/locales/en/translation.json:69]: Should this be uppercase?

**Review by escapedcat — APPROVED** [review]
Please rebase

File: src/i18n/locales/en/translation.json (hunk 1 above)
Comment (bumi) [reply src/i18n/locales/en/translation.json:69]: what? you mean "Numbers..."
I don't know/care.
image

**Comment by bumi** [comment]
now in #1433

**Pull request closed without merging**